dev-infra/commit-message/wizard.ts

```diff
@@ -25,7 +26,12 @@ const defaultCommitMessage = `<type>(<scope>): <summary>
 
 export async function runWizard(
     args: {filePath: string, source?: PrepareCommitMsgHookSource, commitSha?: string}) {
-  // TODO(josephperrott): Add support for skipping wizard with local untracked config file
+  if (getUserConfig().commitMessage?.disableWizard) {
+    debug('Skipping commit message wizard as commitMessage.disableWizard has truthy value in the');
```

Comment:
Might be simpler to say: 
```suggestion
    debug('Skipping commit message wizard due to enabled `commitMessage.disableWizard` option in user config.');
```